specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/managedClusters.json

```diff
@@ -729,6 +729,100 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/agentPools/{agentPoolName}/upgradeProfiles/default": {
```

Comment:
Is this API valid for anything other than "default"?

Reply:
Nope. It is always default for now. This is the same behavior as managedclusters/upgradeProfiles/default.